.github/workflows/embedding-sdk.yml

```diff
@@ -91,6 +91,35 @@ jobs:
       - name: Run frontend embedding SDK snippets type check
         run: yarn run embedding-sdk:docs-snippets:type-check
 
+  embedding-sdk-no-exposed-internal-types-validate:
+    needs: [ files-changed, build ]
+    if: |
+      needs.files-changed.outputs.frontend_embedding_sdk_sources == 'true' &&
+      needs.build.result == 'success'
+    runs-on: ubuntu-22.04
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Prepare front-end environment
+        uses: ./.github/actions/prepare-frontend
+      - name: Retrieve Embedding SDK dist artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: embedding-sdk-${{ github.event.pull_request.head.sha || github.sha }}
+          path: ${{ github.workspace }}/resources/embedding-sdk
+      - name: Run no exposed internal types validation
+        run: yarn run embedding-sdk:docs:generate:pure --emit none
```

Comment:
Is it possible for the docs generation to fail because of another reason other than exposing internal types, e.g. a failure within typedoc config? I wonder if it could potentially be a false positive, and if including the error logs from that command may help.

Reply:
Adjusted a bit to fail only on a specific error message

Reply:
awesome, thanks!